app/utility/RepDataMap.js

```diff
@@ -59,21 +59,21 @@ export const isValidData = (repData) => {
     let initialFlag = parseData(repData, 0);
     console.tron.log("initial flag is " + initialFlag);
 
-    if (initialFlag == -3456) {
+    if (initialFlag == -3456 || !infinityCheck(repData)) {
```

Comment:
- [ ] you shouldn't do the infinity check until the other flags are checked. Like if the initial flag is wrong and the bulk data transmission flag is wrong, then your velocity likely isn't in the right spot. do the check after `if (bulkDataTransmissionFlag == -9999) {`